server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -1402,54 +1402,56 @@ protected void checkIfPodIsDeletable(final long podId) {
         }
     }
 
-    private void checkPodAttributes(final long podId, final String podName, final long zoneId, final String gateway, final String cidr, final String startIp, final String endIp, final String allocationStateStr,
```

Comment:
@shwstppr can you disect this method in smaller parts?

Reply:
done